#使用说明   1.该动能包提供4种多点控制的模式：1.只进行多点规划  3.进行多点规划并且实现每个点的停留时间控制 3.进行多点规划并且实现yaw角控制  4.进行多点规划并且实现每个点的停留时间控制以及yaw角控制
#          2.四种模式中的每个数组元素个数分别要满足3、4、4、5，否则读取失败，以下是满足要求的测试案例。
#          3.启动节点后，在rivz中发送一次2D Nav Goal即可进行规划。

#模式1  x y z 
test1:
  - [-18, 0.0, 1.2]  # Point 1 
  - [0,   8.0, 1.2]    # Point 2   
  - [0,  -2.0, 1.2]   # Point 3

# 模式2 x y z time
test2:
  - [-18, 0.0, 1.2, 2]  # Point 1 
  - [0,   8.0, 1.2, 0]  # Point 2   
  - [0,  -2.0, 1.2, 0] # Point 3

# 模式3 x y z yaw
test3:
  - [-3, 0.0, 1.0, 3]  # Point 1 
  - [0,   8.0, 1.2, 4]  # Point 2   
  - [0,  -2.0, 1.2, 3] # Point 3

# 模式4  x y z yaw time
test4:
  - [-3.0,   0.0,  1.0,  1.57,  0]  # Point 1 
  - [6.5,   0.0,  1.0,  -100,  0]  # Point 2

test_back:
  - [0,   0,   1.0] 
